src/screens/ResultScreen.tsx

```diff
@@ -72,10 +72,18 @@ interface ResultScreenPropsType {
 
 const ResultScreen = ({ route, navigation }: ResultScreenPropsType): JSX.Element => {
   const { extraParams, results, counts, resultType } = route.params
+  const { exercise } = extraParams
   const [entries, setEntries] = React.useState<DocumentResultType[]>([])
   const [isLoading, setIsLoading] = React.useState(true)
   const { Icon, title, order } = resultType
-  const nextResultType = RESULTS.find(result => result.order === (order + 1) % RESULTS.length) ?? RESULTS[0]
+
+  let nextResultType = RESULTS.find(result => result.order === (order + 1) % RESULTS.length) ?? RESULTS[0]
+  if (
+    nextResultType.key === 'similar' &&
+    (exercise === ExerciseKeys.learnArticles || exercise === ExerciseKeys.singleChoice)
+  ) {
+    nextResultType = RESULTS[2]
+  }
```

Comment:
What is this doing? :thinking:

Reply:
For single choice exercises there is no option for "almost correct" results, and therefore they are skipped.